.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Probably still don't want to commit this, given that it's empty you might as well delete it locally too